### PR TITLE
Implement libkylons library

### PR DESCRIPTION
**Pull request by herou**
I have implemented libkylons library and using the execute receipt function of this lib.

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -27,6 +26,21 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
+//    configurations {
```
Comment (MikeSofaer) [inline app/build.gradle]: Are we going to put this back in?

File: app/src/main/java/com/pylons/loud/activities/GameScreenActivity.kt [hunk 2]
```diff
@@ -383,43 +386,29 @@ class GameScreenActivity : AppCompatActivity(),
...
-                )
-
-                CoroutineScope(IO).launch {
-                    val tx = txFlow {
-                        Core.engine.applyRecipe(recipeId, itemIds.toTypedArray())
+                runBlocking {
+                    launch {
+                        WalletInitializer.getWallet().listRecipes {
+                            it.forEach {
+                                recipes.add(it)
```
Comment (MikeSofaer) [inline app/src/main/java/com/pylons/loud/activities/GameScreenActivity.kt:393]: I'm not sure why this is needed

File: app/src/main/java/com/pylons/loud/pylons/ipc/IIpcInterface.java [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+
+        @Override
+        public boolean onTransact(int code, android.os.Parcel data, android.os.Parcel reply, int flags) throws android.os.RemoteException {
+            java.lang.String descriptor = DESCRIPTOR;
+            switch (code) {
+                case INTERFACE_TRANSACTION: {
+                    reply.writeString(descriptor);
+                    return true;
+                }
+                case TRANSACTION_wallet2easel: {
```
Comment (MikeSofaer) [inline app/src/main/java/com/pylons/loud/pylons/ipc/IIpcInterface.java]: Should change these names

**Review by MikeSofaer — COMMENTED** [review]
I would like to see the code we're no longer using removed.

File: app/src/main/java/com/pylons/loud/activities/GameScreenActivity.kt (hunk 2 above)
Comment (herou) [reply app/src/main/java/com/pylons/loud/activities/GameScreenActivity.kt:393]: Actually, we need this, It adds recipes to the list which we need later.

File: app/build.gradle (hunk 1 above)
Comment (herou) [reply app/build.gradle]: I am removing it.

File: app/src/main/java/com/pylons/loud/pylons/ipc/IIpcInterface.java (hunk 3 above)
Comment (herou) [reply app/src/main/java/com/pylons/loud/pylons/ipc/IIpcInterface.java]: The easel word should be replaced with loud.

**Comment by kiroj305** [comment]
Deprecated pr...

**Pull request closed without merging**